spec/requests/api/markers_spec.rb

```diff
@@ -34,10 +34,10 @@
         subject
         expect(json).to include(heading_text: vacancy.job_title)
         expect(json).to include(heading_url: job_path(vacancy))
+        expect(json).to include(name: organisation.name)
         expect(json).not_to include(anonymised_id: StringAnonymiser.new(vacancy.id).to_s)
         expect(json).to include(address: full_address(organisation))
         expect(json).to include(description: nil)
-        expect(json[:details].size).to be 4
```

Comment:
why have you removed this test?

Reply:
i was a little unsure what to do here tbh, but the reason it came about was that the popup data requirements needed to change and one field `phases` was often not present and so i filtered all the details field in the controller so no null values were present. happy to talk about it in the morning, deleting the test was a bit of a copout im sure!